### PR TITLE
Handle No tapes, use RC8 release

### PR DESCRIPTION
**Pull request by berkeleysquare**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -37,7 +37,7 @@ subprojects {
         compile 'ch.qos.logback:logback-core:1.1.6'
         compile 'ch.qos.logback:logback-classic:1.1.6'
         compile 'com.google.guava:guava:19.0'
-        compile 'com.spectralogic.ds3:ds3-sdk:3.0.0-RC7'
+        compile 'com.spectralogic.ds3:ds3-sdk:3.2.0-RC8'
```
Comment (DenverM80) [inline build.gradle]: I don't think we want to move to 3.2 yet, not positive though

Comment (berkeleysquare) [inline build.gradle]: Nossir, that's an oversight.

File: ds3_java_cli/src/main/java/com/spectralogic/ds3cli/views/cli/GetTapesWithFullDetailsView.java [hunk 2]
```diff
@@ -33,7 +33,7 @@
     @Override
     public String render(final GetTapesWithFullDetailsResult obj) throws JsonProcessingException {
         final NamedDetailedTapeList result = obj.getTapesWithDetails();
-        if ((result == null) || (null == result.getNamedDetailedTapes())){
+        if ((result == null) || (null == result.getNamedDetailedTapes()) || (result.getNamedDetailedTapes().size() == 0 ) ){
```
Comment (DenverM80) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/views/cli/GetTapesWithFullDetailsView.java]: the null == in the middle condition sets off my OCD (backwards comparison) - I know you didn't change this... I may have even written as I used to prefer the constant to be on the left in comparisons until the PHB put the kibosh on that

File: ds3_java_cli/src/main/java/com/spectralogic/ds3cli/views/cli/GetTapesWithFullDetailsView.java [hunk 3]
```diff
@@ -33,7 +33,7 @@
     @Override
     public String render(final GetTapesWithFullDetailsResult obj) throws JsonProcessingException {
         final NamedDetailedTapeList result = obj.getTapesWithDetails();
-        if ((result == null) || (null == result.getNamedDetailedTapes())){
+        if ((result == null) || (result.getNamedDetailedTapes() == null) || (result.getNamedDetailedTapes().size() == 0 ) ){
```
Comment (rpmoore) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/views/cli/GetTapesWithFullDetailsView.java]: You should be able to use a Guard method to check if the List returned by getNamedDetailedTapes is null or empty.

**Pull request merged**